openmmtools/tests/test_sampling.py

```diff
@@ -1645,10 +2175,14 @@ class TestReplicaExchange(TestMultiStateSampler):
     def test_stored_properties(self):
         """Test that storage is kept in sync with options. Unique to ReplicaExchange"""
         additional_values = {}
-        additional_values.update(self.property_creator('replica_mixing_scheme', 'replica_mixing_scheme', None, None))
+        additional_values.update(
+            self.property_creator(
+                "replica_mixing_scheme", "replica_mixing_scheme", None, None
+            )
+        )
         self.actual_stored_properties_check(additional_properties=additional_values)
 
-    @attr('slow')  # Skip on Travis-CI
+    @pytest.mark.slow  # Skip on Travis-CI
```

Comment:
Non-blocking comment. Just for future references, there are a few of tests that are marked as slow. We should probably try to reassess if these are still slow with modern computers/CPUs.

Reply:
Created an issue for this as well. https://github.com/choderalab/openmmtools/issues/748